docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -0,0 +1,95 @@
+
+##### docker.release
+Retention Policy: No automatic removal. Requires TSC approval to remove images from this repository.
+
+Contains: Officially released docker images for EdgeX. 
+
+Docker Tags Used:•Version (ie: v1.x), Latest 
+
+[Nexus Cleanup Policies Reference](https://help.sonatype.com/repomanager3/cleanup-policies)
+
+### Docker Compose Files
+
+*Tied to Code Release?* Yes
+
+Docker compose files are released alongside the docker images for every release of EdgeX Foundry. During development the community maintains a compose file named `nightly-build`. This compose file is meant to be a beta version of EdgeX. At the time of release the community makes a named release compose file. (ie: `geneva`)
+
+### Github Page: EdgeX Docs
+
+*Tied to Code Release?* No
+
+EdgeX Foundry releases a set of documentation for our project at http://docs.edgexfoundry.org. This page is a github page that is managed by the `edgex-docs` Github repository. As a community we make our best effort to keep these docs up to date. On this page we are also versioning the docs with the semantic versions of the named releases. As a community we try to version our documentation site shortly after the official release date but documentation changes are addressed as we find them throughout the release cycle.
+
+### GitHub Tags
+
+*Tied to Code Release?* Yes, for the final semantic version
+
+Github tags are used to track the releases of EdgeX Foundry. During development the tags are incremented automatically using a development suffix (ie: `v1.1.1-dev.1` -> `v1.1.1-dev.2`). At the time of release we release a tag with the final semantic version (ie: `v1.1.1`).
+
+### Snaps
+
+*Tied to Code Release?* Yes
+
+Snaps are released for every named release of EdgeX Foundry. During development the community stages snaps to the `latest/edge` channel of the [Snapcraft Store](https://snapcraft.io/search?q=edgexfoundry). At the time of code freeze we will promote the snaps from `latest/edge` to the `latest/beta` amd `latest/candidate` channels as beta release. This beta release is to trigger validation of the release candidate. At the time of release we will promote the snaps from `latest/beta` to `latest/stable`. In addition we also create a named release track for the release. (ie: `geneva/stable`).
+
+### SwaggerHub API Docs
+
+*Tied to Code Release?* No
+
+In addition to our documentation site EdgeX foundry also releases our API specifications on Swaggerhub.
+
+### Testing Framework
+
+*Tied to Code Release?* Yes
+
+The EdgeX Foundry community has a set of tests we maintain to do regression testing during development this framework is tracking the `master` branch of the components of EdgeX. At the time of release we will update the testing frameworks to point at the released Github tags and add a version tag to the testing frameworks themselves. This creates a snapshot of testing framework at the time of release for validation of the official release.
```

Comment:
suggest: ...testing during development**.**

Also I'm not sure what "This framework is tracking the `master` branch of the components of EdgeX" means? I thought that our blackbox tests run against docker images? When we run blackbox tests, we don't rebuild EdgeX and create docker images dynamically.

Reply:
On every merge to master there are docker images created with the docker tag, `master`. The testing frameworks are using these images to test so that they are in sync with the changes in the `master` branch.

Reply:
Thanks for the explanation. Maybe that should get folded into your description here, as it wasn't clear to me?